vowpalwabbit/bs.cc

```diff
@@ -41,9 +41,10 @@ void bs_predict_vote(example& ec, vector<double> &pred_vec)
   // float sum_labels = 0; // uncomment for: "avg on votes" and getLoss()
   bool majority_found = false;
   bool multivote_detected = false; // distinct(votes)>2: used to skip part of the algorithm
-  int* pred_vec_int = new int[pred_vec.size()];
+  auto pred_vec_sz = pred_vec.size();
+  int* pred_vec_int = new int[pred_vec_sz];
```

Comment:
This isn't your fault, but generally speaking reductions should not use 'new' in the middle---we want to externalize memory allocations to reduce overhead.